Mathlib/Probability/Distributions/Gaussian.lean

```diff
@@ -114,6 +121,30 @@ lemma integral_gaussianPdfReal_eq_one (μ : ℝ) {v : ℝ≥0} (hv : v ≠ 0) :
     (ae_of_all _ (gaussianPdfReal_nonneg _ _)), ← ENNReal.ofReal_one] at h
   rwa [← ENNReal.ofReal_eq_ofReal_iff (integral_nonneg (gaussianPdfReal_nonneg _ _)) zero_le_one]
 
+lemma gaussianPdfReal_sub {μ : ℝ} {v : ℝ≥0} (x y : ℝ) :
+    gaussianPdfReal μ v (x - y) = gaussianPdfReal (μ + y) v x := by
+  simp only [gaussianPdfReal]
+  rw [@sub_add_eq_sub_sub_swap]
+
+lemma gaussianPdfReal_inv_mul {μ : ℝ} {v : ℝ≥0} {c : ℝ} (hc : c ≠ 0) (x : ℝ) :
+    gaussianPdfReal μ v (c⁻¹ * x) = |c| * gaussianPdfReal (c * μ) (⟨c^2, sq_nonneg _⟩ * v) x := by
```

Comment:
It might be worth adding another one along the lines of
```lean
lemma gaussianPdfReal_mul {μ : ℝ} {v : ℝ≥0} {c : ℝ} (hc : c ≠ 0) (x : ℝ) :
    gaussianPdfReal μ v (c * x) = |c| * gaussianPdfReal (c⁻¹ * μ) (⟨c^2, sq_nonneg _⟩⁻¹ * v) x := by
```